ui/admin/app/components/auth-and-model-providers/ProviderForm.tsx

```diff
@@ -67,12 +66,18 @@ const translateUserFriendlyLabel = (label: string) => {
 		"OBOT_AUTH_PROVIDER",
 		"OBOT_GOOGLE_AUTH_PROVIDER",
 		"OBOT_GITHUB_AUTH_PROVIDER",
+		"OBOT_GENERIC_OPENAI_MODEL_PROVIDER",
```

Comment:
Looks like we're missing a couple fields here. Can you add them in?
```suggestion
		"OBOT_GENERIC_OPENAI_MODEL_PROVIDER",
    "OBOT_XAI_MODEL_PROVIDER",
    "OBOT_DEEPSEEK_MODEL_PROVIDER",
```

Reply:
Gemini was also missing :thinking: 
Fixed.